bookkeeper-server/src/main/java/org/apache/bookkeeper/bookie/stats/JournalStats.java

```diff
@@ -216,7 +217,7 @@ public JournalStats(StatsLogger statsLogger, final long maxJournalMemoryBytes,
         journalQueueSize = statsLogger.getCounter(BookKeeperServerStats.JOURNAL_QUEUE_SIZE);
         forceWriteQueueSize = statsLogger.getCounter(BookKeeperServerStats.JOURNAL_FORCE_WRITE_QUEUE_SIZE);
         journalCbQueueSize = statsLogger.getCounter(BookKeeperServerStats.JOURNAL_CB_QUEUE_SIZE);
-        cbThreadPoolQueueSize = statsLogger.getCounter(BookKeeperServerStats.CB_THREAD_POOL_QUEUE_SIZE);
+        cbThreadPoolQueueSize = statsLogger.getCounter(CB_THREAD_POOL_QUEUE_SIZE);
```

Comment:
```suggestion
        cbThreadPoolQueueSize = statsLogger.getCounter(BookKeeperServerStats.CB_THREAD_POOL_QUEUE_SIZE);
```

Keep the style with others?